lab_1_classify_by_unigrams/main.py

```diff
@@ -88,6 +173,16 @@ def collect_profiles(paths_to_profiles: list) -> list[dict[str, str | dict[str,
     :paths_to_profiles: a list of strings to the profiles
     :return: a list of loaded profiles
     """
+    if not isinstance(paths_to_profiles, list):
+        return None
+    processed_profiles = []
+    for path in paths_to_profiles:
+        language_profile = load_profile(path)
+        if isinstance(language_profile, dict):
+            processed_profile = preprocess_profile(language_profile)
+            if isinstance(processed_profile, dict):
```

Comment:
To save identation and make it more readable, consider returning `None` as soon as something goes wrong